test/parallel/test-runner-cli.js

```diff
@@ -185,3 +185,20 @@ const testFixtures = fixtures.path('test-runner');
   assert.match(stdout, /# tests 1/);
   assert.match(stdout, /# pass 1/);
 }
+
+{
+  // Use test with --loader and --require.
+  // This case is common since vscode uses --require to load the debugger.
+  const args = ['--no-warnings',
+                '--loader', join(testFixtures, 'esm', 'loader.mjs'),
+                '--test', join(testFixtures, 'index.test.js')];
+  const child = spawnSync(process.execPath, args, {
```

Comment:
Not for this PR, but I’d expect this test and all the ones like it to run faster if you used `spawnPromisified` instead of `spawnSync`?